patches/@rrweb__record@2.0.0-alpha.17.patch

```diff
@@ -1,8 +1,47 @@
 diff --git a/dist/record.js b/dist/record.js
-index 46ec389fefb698243008b39db65470dbdf0a3857..39d9ac821fd84f5a1494ac1e00c8d4e0e58c432a 100644
+index 46ec389fefb698243008b39db65470dbdf0a3857..7741b3c2c43cc868215dbd0ff1785557b158464a 100644
 --- a/dist/record.js
 +++ b/dist/record.js
-@@ -246,6 +246,9 @@ function isCSSImportRule(rule2) {
+@@ -20,6 +20,15 @@ const testableAccessors$1 = {
+   Element: ["shadowRoot", "querySelector", "querySelectorAll"],
+   MutationObserver: []
+ };
++const isFunction = (x) => typeof x === 'function';
++const isNativeFunction = () => isFunction(x) && x.toString().includes("[native code]");
++const isAngularZonePatchedFunction = (x) => {
++  if (!isFunction(x)) {
++      return false;
++  }
++  const prototypeKeys = Object.getOwnPropertyNames(x.prototype || {});
++  return prototypeKeys.some((key) => key.indexOf('__zone'));
++}
```

Comment:
copies these three functions from posthog-js that are already being used in https://github.com/PostHog/posthog-js/pull/1531